apps/hermes-agent/overlays/nishir-tailnet/kustomization.yaml

```
@@ -16,3 +18,10 @@ patches:
       name: hermes-agent
       version: v1
       kind: NetworkPolicy
+patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)